### PR TITLE
FPC 2.6.2: Made it work by fixing recipe and removing errors

### PR DESCRIPTION
**Pull request by lich**

### CODE REVIEW

File: dev-lang/fpc/fpc_bin-2.6.2.recipe [hunk 1]
```diff
@@ -38,9 +67,13 @@ SOURCE_DIR="fpc-2.6.2.i386-haiku"
 BUILD()
 {
 	echo "Please put as prefix: $prefix"
+	#give user time to copy prefix
+	sleep 5s
```
Comment (waddlesplash) [inline dev-lang/fpc/fpc_bin-2.6.2.recipe]: ????

Comment (mmuman) [inline dev-lang/fpc/fpc_bin-2.6.2.recipe]: if the install script reads stdin for input then:
- you should be able to use a heredoc or some pipe to send it anyway,
- it's broken and must be fixed :-D

Comment (lich) [inline dev-lang/fpc/fpc_bin-2.6.2.recipe]: This was there before, I have no idea how to fix it. But I'll try to make it work.

Comment (pulkomandy) [inline dev-lang/fpc/fpc_bin-2.6.2.recipe]: echo $prefix | someOtherCommand

Or for more complex things (waiting for the program to ask for input, etc), you can use the "expect" tool.

**Comment by waddlesplash** [comment]
No response in over five months. Closing.

**Pull request closed without merging**